docs/admin/ldap/configure-project-mapping.md

```diff
@@ -0,0 +1,73 @@
+# Configure LDAP/Kerberos group to project mapping
+ 
+## Introduction
+A group to project mapping allows you to add members of your LDAP group to a project without having to
```

Comment:
This introduction is quite limited. I would extend the introduction by saying that you can specify the role of the member in a project based on the LDAP group as well.
Also I would make clear that with this functionality the project membership is mirrored based on the LDAP group membership. In particular removing a user from the LDAP group will also remove them from the project.